.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test/3rdParty/babylon"]
+	path = test/3rdParty/babylon
+	url = https://github.com/babel/babylon.git
```

Comment:
@markelog we can't use npm because it will always respect the package.json's name, and files attributes which means it will put a package stripped of tests/fixtures into 'node_modules/babylon'